_includes/post-macros.html

```diff
@@ -298,7 +298,7 @@ <h1 class="summary_heading">{{ post.title|safe }}</h1>
 <div class="expandable
             expandable__filters
             expandable__padded
-            {{ 'expandable__expanded' if is_expanded }}
+            {{ 'expandable__expanded' if is_expanded else ''}}
```

Comment:
Very minor, but the trailing space is missing in all the changes in this file (they are being added in, e.g. _includes/macros/util/format.html).